docs/pages/management/operations/db-ca-migrations.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Database CA Migrations
+description: How to complete Teleport Database CA migrations.
+---
+
+In Teleport Database Access, self-hosted databases must be configured with 
+certificates to enable mTLS authentication via the Teleport Database Service.
+
+Teleport 15 introduced a new `db_client` certificate authority (CA) to split the
+responsibilities of the Teleport `db` CA, which was acting as both host and 
+client CA for Teleport self-hosted database access.
+
+Teleport's host/client database CA split is intended to limit the potential for
+lateral movement to other resources in the event that a database instance's
+private key is compromised.
+
+The `db` and `db_client` CAs were both introduced as an automatic migration
+after upgrading to Teleport 10 and Teleport 15, respectively.
```

Comment:
Since Teleport 10 is out of support, should we mention it on this page? We typically don't talk about feature availability for unsupported versions, though there are cases where significant numbers of users are on an old version and need the migration instructions.

Reply:
In this case, I think we need to mention it.

If a customer has a cluster that was upgraded v9->v10 in the past, and now that cluster has been upgraded all the way to even v15, they still might need to complete that v10 migration by rotating CAs.
We didn't push people too much to rotate their DB CA in v10 because it wasn't introduced as a security fix, whereas now we know that they should rotate for security.
It's very likely that we have customers on supported versions of Teleport who have never rotated their database CA

Reply:
Makes sense, thanks!